lib/validator/v3/validator.go

```diff
@@ -545,6 +545,25 @@ func validateFelixConfigSpec(v *validator.Validate, structLevel *validator.Struc
 			}
 		}
 	}
+
+	// Validate that the externalNodesCIDRList is composed of valid cidr's.
+	if c.ExternalNodesCIDRList != nil {
+		if len(*c.ExternalNodesCIDRList) == 0 {
+			structLevel.ReportError(reflect.ValueOf(c.ExternalNodesCIDRList),
```

Comment:
Do we error on empty slices elsewhere?  Feels like an empty slice should mean "don't add any external CIDRs"